src/FiniteDifferenceWeights.hpp

```diff
@@ -0,0 +1,44 @@
+#include <array>
+#include <type_traits>
+
+template <size_t StencilSize>
+constexpr std::array<std::array<Real, StencilSize>, 6>  fdw()
+{
+    static_assert(StencilSize == 5 || StencilSize == 7 || StencilSize == 9,
+                  "Unsupported stencil size — only 5, 7 or 9 are allowed.");
+
+    if constexpr (StencilSize == 5) {
+        // 5-point stencil, O(h⁴)
```

Comment:
I would avoid non-ASCII characters in source files. Should work mostly, especially in a comment, but it's just asking for incompatibilities.

Reply:
```suggestion
        // 5-point stencil, O(h^4)
```